### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rbellary-vi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Netuitive Rails Agent
 
 The Netuitive Rails Agent creates default rails metrics to be sent to NetuitiveD using the netuitive_ruby_api gem. The Netuitive Rails Agent is meant to work in conjunction with the [netuitive_ruby_api](https://rubygems.org/gems/netuitive_ruby_api) gem and NetuitiveD to help [Netuitive](https://www.netuitive.com) monitor your Ruby applications.
 
-For more information on the Netuitive Rails Agent, see our Ruby agent [help docs](https://help.netuitive.com/Content/Misc/Datasources/new_ruby_datasource.htm), or contact Netuitive support at [support@netuitive.com](mailto:support@netuitive.com).
+For more information on the Netuitive Rails Agent, see our Ruby agent [help docs](https://docs.virtana.com/en/ruby-agent.html), or contact Netuitive support at [support@netuitive.com](mailto:support@netuitive.com).
```
Comment (TheConnMan) [inline README.md:6]: Support link.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ Netuitive Rails Agent
 
 The Netuitive Rails Agent creates default rails metrics to be sent to NetuitiveD using the netuitive_ruby_api gem. The Netuitive Rails Agent is meant to work in conjunction with the [netuitive_ruby_api](https://rubygems.org/gems/netuitive_ruby_api) gem and NetuitiveD to help [Netuitive](https://www.netuitive.com) monitor your Ruby applications.
```
Comment (TheConnMan) [inline README.md:4]: netuitive.com

**Review by TheConnMan — CHANGES_REQUESTED** [review]
PR into `develop`, please.

File: README.md (hunk 2 above)
Comment (rbellary-vi) [reply README.md:4]: changes made

File: README.md (hunk 1 above)
Comment (rbellary-vi) [reply README.md:6]: changes made

Comment (TheConnMan) [reply README.md:6]: I don't see them yet. Did you push?

**Pull request closed without merging**